src/adapters/.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+  "rules": {
+    "import/no-extraneous-dependencies": 0,
+    "import/no-unresolved": 0,
+    "import/extensions": 0,
+    "react/no-deprecated": 0,
+    "react/no-find-dom-node": 0,
+    "react/no-multi-comp": 0,
+    "no-underscore-dangle": 0,
+    "class-methods-use-this": 0
```

Comment:
presumably we can work to remove these overrides when we restructure the adapters?